test/unit/lifecycle/properties.js

```diff
@@ -325,4 +338,21 @@ describe('lifecycle/property', function () {
       });
     });
   });
+  
+  describe('patterns', function () {
```

Comment:
That's already tested here https://github.com/skatejs/skatejs/pull/458/files#diff-c58358d2c478f1fec77c97e11121e54fR211.